File: PhonoArk/PhonoArk/Models/ExamQuestionAttempt.cs
using System;

namespace PhonoArk.Models;

public class ExamQuestionAttempt
{
    public int Id { get; set; }
    public int ExamResultId { get; set; }
    public int QuestionOrder { get; set; }
    public DateTime ExamDate { get; set; }

    public string PhonemeSymbol { get; set; } = string.Empty;
    public string CorrectWord { get; set; } = string.Empty;
    public string CorrectIpa { get; set; } = string.Empty;
    public string UserWord { get; set; } = string.Empty;
    public string UserIpa { get; set; } = string.Empty;
    public bool IsCorrect { get; set; }
}

File: pureopus/app/src/main/java/com/phonoark/data/model/ExamQuestionAttempt.kt
package com.phonoark.data.model

data class ExamQuestionAttempt(
    val id: Long = 0,
    val examResultId: Long = 0,
    val questionOrder: Int,
    val phonemeSymbol: String,
    val correctWord: String,
    val correctWordIpa: String,
    val userAnswerWord: String,
    val userAnswerIpa: String,
    val isCorrect: Boolean
)
